llvm/include/llvm/Transforms/InstCombine/InstCombine.h

```diff
@@ -43,12 +43,20 @@ struct InstCombineOptions {
     MaxIterations = Value;
     return *this;
   }
+
+  /// Only enable skipping in standard optimization pipeline.
+  bool isCompatibleWith(const InstCombineOptions &LastOption) const {
+    return !VerifyFixpoint && !LastOption.VerifyFixpoint &&
+           MaxIterations == InstCombineDefaultMaxIterations &&
+           LastOption.MaxIterations == InstCombineDefaultMaxIterations;
```

Comment:
Shouldn't it just be `VerifyFixpoint == LastOption.VerifyFixPoint && MaxIterations == LastOption.MaxIteration`?

Reply:
Done.